app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/DeviceCommunicationService.java

```diff
@@ -174,7 +174,7 @@ public void onReceive(Context context, Intent intent) {
                         try (DBHandler dbHandler = GBApplication.acquireDB()) {
                             DaoSession session = dbHandler.getDaoSession();
                             boolean askForDBMigration = false;
-                            if (DBHelper.findDevice(device, session) == null && device.getType() != DeviceType.VIBRATISSIMO) {
+                            if (DBHelper.findDevice(device, session) == null && device.getType() != DeviceType.VIBRATISSIMO && (device.getType() != DeviceType.LIVEVIEW)) {
```

Comment:
To me this looks like we should rather check for the old devices for which me might need to do a db migration instead of blacklisting all new devices.